### ISSUE TITLE
Setup Release please

### ISSUE THREAD

**Issue by pascalwilbrink** [com-pas/compas-core#300]
For automatic releasing, we should setup release-please GH Action.

We're currently waiting for V4 of release-please (https://github.com/google-github-actions/release-please-action/pull/883) to be released, since we're making use of a develop/main branch.

Steps to take:

Create release-please.yaml with release-please@v4
Should be triggered on push to `main`.
Target branch should be `main`.

This is the current workflow:
```mermaid
gitGraph LR:
    commit
    branch "Develop"
    branch "Feature1"
    commit
    checkout "Develop"
    commit
    merge "Feature1"
    branch "Feature2"
    commit
    commit
    checkout "Develop"
    merge "Feature2"
    checkout "main"
    merge "Develop"
    branch "Release PR"
    commit id: "Auto generated by Release Please"
    checkout "main"
    merge "Release PR" tag: "Release"
```

Once Release-please is setup correctly and releasing works, we can remove the GH Action `release-project.yml` and move it into `release-please.yml`

**Comment by Sander3003** [comment]
@trusz  @clepski can this one be closed?

**Comment by clepski** [comment]
No, for this repository the release please switch has not been done yet
